src/test/java/io/vertx/kafka/client/tests/ExampleTest.java

```diff
@@ -20,6 +21,7 @@
 import java.util.Map;
 
 @RunWith(VertxUnitRunner.class)
+@Ignore("This is an example test class.")
```

Comment:
what is this example class test for ?

Reply:
I think this is just a reference about how to implement tests. Because there is no boostrap.servers defined, this causes master branch to fail.